src/simplejavatexteditor/Find.java

```diff
@@ -175,7 +175,7 @@ public void replace() {
     }
 
     public void replaceAll() {
-        txt.setText(txt.getText().toLowerCase().replaceAll(textF.getText().toLowerCase(), textR.getText()));
+        txt.setText(txt.getText().replaceAll(textF.getText() , textR.getText()));
```

Comment:
Was there a reason why you removed the case-insensitive`selection.toLowerCase`?

Reply:
Because there is a bug in the replaceAll() method of Find.java class. If we replace the word of Text in NotePad with a word, all the characters who're in Uppercase in NotePad, changed to LowerCase.

Like, let's say there is "Talha is Here" in notePad and I replaced the name "Talha" with "Ali" by using the replace all field.
That statement will first change the text of NotePad "Talha is Here" into "talha is ali".

All Uppercase letters are converted into lower case. 

You can Test that replaceAll() function.